integration-test/src/test/java/com/datastrato/gravitino/integration/test/client/CatalogIT.java

```diff
@@ -0,0 +1,135 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Tag("gravitino-docker-it")
+public class CatalogIT extends AbstractIT {
+
+  private static final Logger LOG = LoggerFactory.getLogger(CatalogIT.class);
+
+  private static final ContainerSuite containerSuite = ContainerSuite.getInstance();
+
+  private static final String metalakeName = GravitinoITUtils.genRandomName("catalog_it_metalake");
+
+  private static GravitinoMetalake metalake;
+
+  private static String hmsUri;
+
+  @BeforeAll
+  public static void startUp() {
+    containerSuite.startHiveContainer();
+    hmsUri =
+        String.format(
+            "thrift://%s:%d",
+            containerSuite.getHiveContainer().getContainerIpAddress(),
+            HiveContainer.HIVE_METASTORE_PORT);
+
+    NameIdentifier ident = NameIdentifier.of(metalakeName);
+    Assertions.assertFalse(client.metalakeExists(ident));
+    metalake = client.createMetalake(ident, "metalake comment", Collections.emptyMap());
+    Assertions.assertTrue(client.metalakeExists(ident));
+  }
+
+  @AfterAll
+  public static void tearDown() {
+    client.dropMetalake(NameIdentifier.of(metalakeName));
+
+    if (client != null) {
+      client.close();
+      client = null;
+    }
+
+    try {
+      closer.close();
+    } catch (Exception e) {
+      LOG.error("Exception in closing CloseableGroup", e);
+    }
+  }
+
+  @Test
+  public void testCreateCatalog() {
+    String catalogName = GravitinoITUtils.genRandomName("catalog");
+    NameIdentifier catalogIdent = NameIdentifier.of(metalakeName, catalogName);
+    Assertions.assertFalse(metalake.catalogExists(catalogIdent));
+
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("metastore.uris", hmsUri);
+    Catalog catalog =
+        metalake.createCatalog(
+            catalogIdent, Catalog.Type.RELATIONAL, "hive", "catalog comment", properties);
+    Assertions.assertTrue(metalake.catalogExists(catalogIdent));
+
+    Assertions.assertEquals(catalogName, catalog.name());
+    Assertions.assertEquals(Catalog.Type.RELATIONAL, catalog.type());
+    Assertions.assertEquals("hive", catalog.provider());
+    Assertions.assertEquals("catalog comment", catalog.comment());
+    Assertions.assertTrue(catalog.properties().containsKey("metastore.uris"));
+
+    metalake.dropCatalog(catalogIdent);
+  }
+
+  @Test
+  @DisabledIfSystemProperty(named = "testMode", matches = "embedded")
+  public void testCreateCatalogWithPackage() {
+    String catalogName = GravitinoITUtils.genRandomName("catalog");
+    NameIdentifier catalogIdent = NameIdentifier.of(metalakeName, catalogName);
+    Assertions.assertFalse(metalake.catalogExists(catalogIdent));
+
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("metastore.uris", hmsUri);
+
+    String gravitinoHome = System.getenv("GRAVITINO_HOME");
+    Assertions.assertNotNull(gravitinoHome);
+    String packagePath = String.join(File.separator, gravitinoHome, "catalogs", "hive");
```

Comment:
Can you move the directory to another directory and test it again as it's still under the folder `gravitinoHome`.

Reply:
It's hard to move the directory to another place. I cannot figure out a better way in test. Actually, by specifying `package`, Gravitino will not fall back to using the default package path, so it can also be shown that `package` is worked.